src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -1,48 +1,60 @@
 import * as React from 'react';
 import { useEffect, useState } from 'react';
-import { getSellers } from '../../utilities';
-import NavBar from './MerchantNavBar';
-import MerchantCard from './MerchantCard';
-import MerchantDescriptionBanner from './MerchantDescriptionBanner';
+import ReactPixel from 'react-facebook-pixel';
+import { useTranslation } from 'react-i18next';
+
 import ContributionBar from './ContributionBar';
-import styles from './styles.module.scss';
-import { LoaderFillerContainer } from '../Loader';
 import DonationHighlightBox from './DonationHighlightBox';
 import GiftMealHighlightBox from './GiftMealHighlightBox';
+import MerchantCard from './MerchantCard';
+import MerchantDescriptionBanner from './MerchantDescriptionBanner';
+import NavBar from './MerchantNavBar';
+import { LoaderFillerContainer } from '../Loader';
+import { getSellers } from '../../utilities';
+import type { BrowsePageSeller } from '../../utilities/api/types';
 import { getWebsiteImages } from '../../utilities/general/StoreImages';
-import { useTranslation } from 'react-i18next';
-import ReactPixel from 'react-facebook-pixel';
+
+import styles from './styles.module.scss';
 
 interface Props {
   menuOpen: boolean;
 }
 
+type ContributionsType = {
+  donationAmount: number,
+  giftCardAmount: number,
+  giftAMealAmount: number,
+}
+
+const INITIAL_CONTRIBUTIONS: ContributionsType = {
+  donationAmount: 0,
+  giftCardAmount: 0,
+  giftAMealAmount: 0,
+};
+
 ReactPixel.trackCustom('MerchantsPageView', {});
 const MerchantsPage = (props: Props) => {
   const websiteImages = getWebsiteImages();
   const { t, i18n } = useTranslation();
   const [sellers, setSellers] = useState<any | null>();
   const [filter, setFilter] = useState<any | null>();
-  const [totalDonations, setDonations] = useState(0);
-  const [totalGiftCards, setGiftCards] = useState(0);
+  const [contributions, setContributions] = useState<ContributionsType>(INITIAL_CONTRIBUTIONS);
 
-  const fetchData = async (lang?) => {
+  const fetchData = async (lang?: string) => {
     const { data } = await getSellers(lang);
 
-    const contributions = data.reduce(
-      (total: any, store: any) => {
-        return [
-          total[0] + store.donation_amount,
-          total[1] + store.gift_card_amount,
-        ];
-      },
-      [0, 0]
+    const tempContributions = data.reduce(
```

Comment:
is this still "temp" or is this more of a "real" value now?

Reply:
ah true, i just named it temp because i didn't want to collide w/ the contributions variable above. i ended up just inlining this